lib/utilities/package-cache.js

```diff
@@ -0,0 +1,385 @@
+var fs = require('fs-extra');
+var path = require('path');
+var quickTemp = require('quick-temp');
+var merge = require('ember-cli-lodash-subset').merge;
+var Configstore = require('configstore');
+var CommandGenerator = require('./command-generator');
+
+/**
+ * The `bower` command helper.
+ *
+ * @method bower
+ * @param {String} subcommand The subcommand to be passed into bower.
+ * @param {String} [...arguments] Arguments to be passed into the bower subcommand.
+ * @param {Object} [options={}] The options passed into child_process.execSync.
+ *   (https://nodejs.org/api/child_process.html#child_process_child_process_execsync_command_options)
+ */
+var bower = new CommandGenerator(require.resolve('bower/bin/bower'), {
```

Comment:
Generator made me think of blueprints + generation, thoughts on renaming this to `CommandBuilder` or `CommandFactory` or ..